src/apps/deskflow-gui/dialogs/AboutDialog.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Deskflow -- mouse and keyboard sharing utility
- * Copyright (C) 2024 Chris Rizzitello <sithlord48@gmail.com>
+ * Copyright (C) 2024 - 2025 Chris Rizzitello <sithlord48@gmail.com>
```

Comment:
Copyright protection begins automatically upon creation of the work, and there is no strict legal requirement to update the copyright year for it to remain valid.

```suggestion
 * Copyright (C) 2024 Chris Rizzitello <sithlord48@gmail.com>
```

Reply:
May not be needed but its a common practice to update when you do new work in a new year

Reply:
Well we'd better update the copyright for Deskflow then too.

![image](https://github.com/user-attachments/assets/093be686-b7ee-4287-bd4b-fc7fb9b308eb)

Reply:
its in the use cmake var for copyright PR